src/strings/en-us.json

```diff
@@ -1082,8 +1082,6 @@
     "MessageContactAdminToResetPassword": "Please contact your system administrator to reset your password.",
     "MessageCreateAccountAt": "Create an account at {0}",
     "MessageDeleteTaskTrigger": "Are you sure you wish to delete this task trigger?",
-    "MessageDirectoryPickerBSDInstruction": "For BSD, you may need to set up storage within your 'FreeNAS Jail' so Jellyfin can access your media.",
-    "MessageDirectoryPickerLinuxInstruction": "For Linux on Arch Linux, CentOS, Debian, Fedora, openSUSE, or Ubuntu, you must grant the service user at least read access to your storage locations.",
```

Comment:
Now that I see what these help messages should've shown, it might be worth combining these two texts into one and always displaying that? Kinda useful to have in-app help.

Reply:
Good idea.

Reply:
Maybe we can have a small text and link to a (new) documentation page with a more extensive explanation?